overrides/app/pages/algolia-product-list/partials/algolia-color-refinements.jsx

```diff
@@ -15,61 +27,97 @@ const AlgoliaColorRefinements = (props) => {
     })
 
     const {items, refine} = useRefinementList(props)
+    const [visibleItemsCount, setVisibleItemsCount] = useState(6)
+
+    const handleSeeMore = () => {
+        setVisibleItemsCount((prevCount) => prevCount + 6)
+    }
 
     return (
-        <AlgoliaRefinementsContainer title={props.title} attributes={[props.attribute]}>
-            <SimpleGrid columns={2} spacing={2} mt={1}>
-                {items.map((item, idx) => {
-                    const lcLabel = item.label.toLowerCase().replace(/\s/g, '').replace(/&/g, 'and')
-                    return (
-                        <Box key={idx}>
-                            <HStack onClick={() => refine(item.label)} spacing={1} cursor="pointer">
-                                <Button
-                                    {...styles.swatch}
-                                    color={item.isRefined ? 'black' : 'gray.200'}
-                                    aria-checked={item.isRefined}
-                                    variant="outline"
-                                    marginRight={0}
-                                    marginBottom="-1px"
-                                    width="20px"
-                                    height="20px"
-                                    borderRadius="100%"
-                                    overflow="hidden"
-                                    minWidth="auto"
-                                    border={'1px solid #e9e9e9'}
+        <>
+            {items.length > 0 && (
+                <AccordionItem>
+                    <h2>
+                        <AccordionButton>
+                            <Box as="span" flex="1" textAlign="left">
+                                {props.title}
+                            </Box>
+                            <AccordionIcon />
+                        </AccordionButton>
+                    </h2>
+                    <AccordionPanel pb={4}>
+                        <AlgoliaRefinementsContainer attributes={[props.attribute]}>
+                            <SimpleGrid columns={1} spacing={2} mt={1}>
+                                {items.slice(0, visibleItemsCount).map((item, idx) => {
```

Comment:
Did you try the beta `ColorRefinementList` widget? https://www.algolia.com/doc/api-reference/widgets/color-refinement-list/react/
What's missing from it?

Reply:
We are using React InstantSearch version 7 and it is only available for version 6

Reply:
Oh, indeed :+1: